.travis.yml

```diff
@@ -10,7 +11,7 @@ matrix:
     - python: '3.3'
```

Comment:
Just as a canary in the coal mine. We were testing pypy and pypy3, too, until they produced too many false positives.